packages/admin-frontend/pages/featured.js

```diff
@@ -0,0 +1,317 @@
+          validate={handleValidate}
+          render={({ handleSubmit, pristine, invalid }) => (
+            <form>
+              <Grid container spacing={24} direction="column">
+                <Grid item xs>
+                  <Field
+                    name="title"
+                    render={({ input, meta }) => (
+                      <div>
+                        <TextField
+                          fullWidth
+                          error={meta.error && meta.touched}
+                          required
+                          disabled={this.state.selectedLanguage === ''}
+                          label="Title"
+                          {...input}
+                        />
+                        {meta.error &&
+                          meta.touched && (
+                            <FormHelperText error>{meta.error}</FormHelperText>
+                          )}
+                      </div>
+                    )}
+                  />
+                </Grid>
+
+                <Grid item xs>
+                  <Field
+                    name="description"
+                    render={({ input, meta }) => (
+                      <div>
+                        <TextField
+                          fullWidth
+                          required
+                          error={meta.error && meta.touched}
+                          disabled={this.state.selectedLanguage === ''}
+                          label="Description"
+                          {...input}
+                          multiline
+                        />
+                        {meta.error &&
+                          meta.touched && (
+                            <FormHelperText error>{meta.error}</FormHelperText>
+                          )}
+                      </div>
+                    )}
+                  />
+                </Grid>
+
+                <Grid item xs>
+                  <Field
+                    name="link"
+                    render={({ input, meta }) => (
+                      <div>
+                        <TextField
+                          fullWidth
+                          type="url"
+                          error={meta.error && meta.touched}
+                          required
+                          disabled={this.state.selectedLanguage === ''}
+                          label="Link"
+                          {...input}
+                        />
+                        {meta.error &&
+                          meta.touched && (
+                            <FormHelperText error>{meta.error}</FormHelperText>
+                          )}
+                      </div>
+                    )}
+                  >
+                    />
+                  </Field>
+                </Grid>
+
+                <Grid item xs>
+                  <Field
+                    name="imageUrl"
+                    render={({ input, meta }) => (
+                      <div>
```

Comment:
Can we use JSX fragments instead of the wrapping divs here and for the other fields? Especially if the div is just there to group the children together